v0/destinations/splitio/data/EventConfig.json

```diff
@@ -0,0 +1,78 @@
+[
+  {
+    "destKey": "eventTypeId",
+    "sourceKeys": [
+      "eventTypeId",
+      "properties.eventTypeId",
+      "event",
+      "name",
+      "traits.eventTypeId",
+      "context.traits.eventTypeId"
+    ],
+    "metadata": {
+      "type": "toString"
+    },
+    "required": true
+  },
+  {
+    "destKey": "environmentName",
+    "sourceKeys": [
+      "environmentName",
+      "properties.environmentName",
+      "traits.environmentName",
+      "context.traits.environmentName"
+    ],
+    "metadata": {
+      "type": "toString"
+    },
+    "required": false
+  },
+  {
+    "destKey": "trafficTypeName",
+    "sourceKeys": [
+      "trafficTypeName",
+      "properties.trafficTypeName",
+      "traits.trafficTypeName",
+      "context.traits.trafficTypeName"
+    ],
+    "required": true
+  },
+  {
+    "destKey": "key",
+    "sourceKeys": [
+      "key",
+      "properties.key",
+      "traits.key",
+      "context.traits.key",
+      "userId",
+      "anonymousId"
+    ],
+    "required": true
+  },
+  {
+    "destKey": "timestamp",
+    "sourceKeys": [
+      "properties.timestamp",
+      "traits.timestamp",
+      "context.traits.timestamp",
+      "timestamp"
+    ],
+    "metadata": {
+      "type": "secondTimestamp"
+    },
+    "required": false
+  },
+  {
+    "destKey": "value",
+    "sourceKeys": [
+      "value",
```

Comment:
values can be mapped from revenue > value > total from properties/traits

Reply:
added only from properties, as per description .